tests/unit/customizations/test_paginate.py

```diff
@@ -54,6 +54,8 @@ def test_customize_arg_table(self):
             'foo': mock.Mock(),
             'bar': mock.Mock(),
         }
+        # Default to false
+        argument_table['foo']._UNDOCUMENTED = False
```

Comment:
Why was this line needed to be added? Especially since it was not done for ``'bar'``.